core/state_prefetcher.go

```diff
@@ -65,7 +65,7 @@ func (p *statePrefetcher) Prefetch(block *types.Block, statedb *state.StateDB, c
 			return // Also invalid block, bail out
 		}
 		statedb.SetTxContext(tx.Hash(), i)
-		if err := precacheTransaction(msg, gaspool, evm); err != nil {
+		if _, err := ApplyMessage(evm, msg, gaspool); err != nil {
```

Comment:
I think it would be good to add a comment here e.g.
```
// We attempt to apply a transactio. The goal is not to execute
// the transaction successfully, rather to warm up touched data slots.
```